python/src/aac/validate/_validate.py

```diff
@@ -78,6 +78,9 @@ def _with_validation(
         result = _validate_definitions(user_definitions, language_context, validate_context)
 
         if result.is_valid():
+            print("The following file(s) have been validated:")
```

Comment:
While I am fairly confident this gets us the information we wanted with this task, I am not sure that just printing is what was desired.